lib/features/email/presentation/extensions/calendar_event_extension.dart

```diff
@@ -356,6 +356,7 @@ extension CalendarEventExtension on CalendarEvent {
   }
 
   bool get isDisplayedEventReplyAction => method != null
+    && method != EventMethod.reply
```

Comment:
While we are at it, what are other possible values here and for which others does it make sense to hide the buttuns?

Reply:
Other possible values

![Screenshot 2024-10-24 at 09 16 13](https://github.com/user-attachments/assets/f85d3166-aa14-4623-83ce-37e3ff3f71c9)

Reply:
IMO

Publish: Hide
Request: Show
Reply: Hide
Add: Show
Cancel: Hide
Refresh: Hide
Counter: Show
DeclineCounter: Hide

@chibenwa How about you?

Reply:
Agree